### PR TITLE
add sleep command

### PR DESCRIPTION
**Pull request by atakanyenel**
I loved this project so I added a new command `sleep` that sleeps for given number of seconds. I will also work on a command that allows to download additional compiled plugins from github to the shell, similar to `apt-get install`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ files that implement command plugins.
 #### Pre-requisites
 
 * Go 1.8 or above
-* Linux (for now)
+* Linux
+* Mac OSX
```
Comment (gayanhewa) [inline README.md]: Full compatibility with osx?

Comment (atakanyenel) [reply README.md]: I developed it on an OSX machine, so I expect full compatibility so far.

**Comment by vladimirvivien** [comment]
Apologies I missed this PR.

**Comment by vladimirvivien** [comment]
LGTM

**Pull request merged**